From: Colin Ian King <colin.king@canonical.com>
Date: Fri, 14 Feb 2020 00:33:38 +0000
Subject: RDMA/hns: fix spelling mistake: "attatch" -> "attach"
Patch-mainline: v5.7-rc1
Git-commit: 8d8d2b76ac3545cf6c0e9c6aa61700db98d2496a
References: jsc#SLE-14777

There is a spelling mistake in a dev_err error message. Fix it.

Link: https://lore.kernel.org/r/20200214003338.6573-1-colin.king@canonical.com
Signed-off-by: Colin Ian King <colin.king@canonical.com>
Signed-off-by: Jason Gunthorpe <jgg@mellanox.com>
Acked-by: Thomas Bogendoerfer <tbogendoerfer@suse.de>
---
 drivers/infiniband/hw/hns/hns_roce_hw_v2.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/infiniband/hw/hns/hns_roce_hw_v2.c
+++ b/drivers/infiniband/hw/hns/hns_roce_hw_v2.c
@@ -5783,7 +5783,7 @@ static int map_eq_buf(struct hns_roce_de
 			  page_shift);
 	ret = hns_roce_mtr_attach(hr_dev, &eq->mtr, &buf_list, &region, 1);
 	if (ret)
-		dev_err(hr_dev->dev, "mtr attatch error for eqe\n");
+		dev_err(hr_dev->dev, "mtr attach error for eqe\n");
 
 	goto done;
 
